### PR TITLE
[wasm] Attempt to fix audit errors in sanitize

### PR DESCRIPTION
**Pull request by lewing**

### CODE REVIEW

File: src/mono/wasm/sanitize.py [hunk 1]
```diff
@@ -64,9 +64,19 @@ def prune():
         print(error)
 
 
+for path in node_paths:
+    if upgrade:
+        update_npm(path)
```
Comment (radical) [inline src/mono/wasm/sanitize.py:69]:
```suggestion
if upgrade:
    for path in node_paths:
        update_npm(path)
```

File: src/mono/wasm/sanitize.py [hunk 2]
```diff
@@ -64,9 +64,19 @@ def prune():
...
+
```
Comment (radical) [inline src/mono/wasm/sanitize.py:70]:
```suggestion
```

File: src/mono/wasm/sanitize.py [hunk 3]
```diff
@@ -85,7 +95,5 @@ def prune():
 prune()
 
 for path in node_paths:
-    if upgrade:
-        update_npm(path)
-    else:
+    if not upgrade:
```
Comment (radical) [inline src/mono/wasm/sanitize.py:98]: nit: move outside the loop

File: src/mono/wasm/sanitize.py (hunk 2 above)
Comment (lewing) [reply src/mono/wasm/sanitize.py:70]: makes the linter angry

**Comment by lewing** [comment]
Failures are unrelated, this target doesn't even run on PRs but is used by the perf lane for wasm

**Review by steveisok — APPROVED** [review]

**Pull request merged**